benchmark/Microsoft.IdentityModel.Benchmarks/ReadTokenTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using BenchmarkDotNet.Attributes;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Microsoft.IdentityModel.Benchmarks
+{
+    // dotnet run -c release -f net8.0 --filter Microsoft.IdentityModel.Benchmarks.ReadTokenTests*
+
+    [Config(typeof(BenchmarkConfig))]
+    [HideColumns("Type", "Job", "WarmupCount", "LaunchCount")]
+    [MemoryDiagnoser]
+    public class ReadTokenTests
+    {
+        string _encodedJWS;
+        string _encryptedJWE;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            var jsonWebTokenHandler = new JsonWebTokenHandler();
+            var jwsTokenDescriptor = new SecurityTokenDescriptor
+            {
+                SigningCredentials = BenchmarkUtils.SigningCredentialsRsaSha256,
+                TokenType = JwtHeaderParameterNames.Jwk,
+                Claims = BenchmarkUtils.Claims
+            };
+
+            _encodedJWS = jsonWebTokenHandler.CreateToken(jwsTokenDescriptor);
+
+            var jweTokenDescriptor = new SecurityTokenDescriptor
+            {
+                SigningCredentials = BenchmarkUtils.SigningCredentialsRsaSha256,
+                EncryptingCredentials = BenchmarkUtils.EncryptingCredentialsAes256Sha512,
+                TokenType = JwtHeaderParameterNames.Jwk,
+                Claims = BenchmarkUtils.Claims
+            };
+
+            _encryptedJWE = jsonWebTokenHandler.CreateToken(jweTokenDescriptor);
+        }
+
+        [Benchmark]
```

Comment:
```suggestion
        [Benchmark(Baseline = true)]
```

Reply:
The same [ReadTokens(..)](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/d5744855dcf94816a970dae70b99dcc82ffe324c/src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs#L410C9-L410C73) method is invoked by all the JsonWebToken constructors. Making the string constructor the baseline and ranking it may not make sense.

The goal here is to maintain a single code path and ensure the existing constructors that accept a string are able to take advantage of the performance improvements from using spans.